training/trillium/GPT3-175B-MaxText/README.md

```diff
@@ -0,0 +1,30 @@
+# Instructions for training GPT3-175B-Maxtext on TPU trillium
+
+## XPK setup
+Please follow this [link](https://github.com/AI-Hypercomputer/tpu-recipes/training/trillium/XPK_README.md) to create your GKE cluster with XPK
+
+## Prep for Maxtext 
+Please follow this [link](https://github.com/AI-Hypercomputer/tpu-recipes/training/trillium/MAXTEXT_README.md) to install maxtext and build docker image
+
+## Run Maxtext GPT3-175B workloads on GKE
+
+### Test Env
+jaxlib=0.4. 35
+
+libtpu-nighly=20241028
```

Comment:
@suexu1025 The libtpu version is the one that is in the JAX stable stack (which is derived from the JAX version). Is there a specific reason we are mentioning this here and in other files that this is the test environment? I thought the mention of overriding the libtpu version to nightly was to get access to the latest and experimental features.

Reply:
stable stack used libtpu`0.1.dev20241010+nightly.cleanup` and we noticed non-trival performance improvement using this newer libtpu and matching with launch pad.